prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/KS/KS-receiver-transform.yml

```diff
@@ -0,0 +1,59 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+elements:
+
+  - name: xtn-use-code-from-system
+    condition: '%resource.system = "email"'
+    value: [ '"NET"' ]
+    hl7Spec: [ '%{hl7TelecomPath}-2' ]
+
+  - name: ks-message-encoding-characters
+    condition: true
+    value: [ '"^~\\&"' ]
```

Comment:
Why is this needed? All message should have this encoding characters by default. We only need to override it if it's something different.

Reply:
Ok, it will be removed.